Packs/CyberArkPAS/Integrations/CyberArkPAS/README.md

```diff
@@ -0,0 +1,1804 @@
+Use the CyberArk Privileged Access Security (PAS) solution to manage users, safes, vaults, and accounts from Cortex XSOAR.
+This integration was integrated and tested with version xx of CyberArkPAS
+## Configure CyberArkPAS on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for CyberArkPAS.
+3. Click **Add instance** to create and configure a new integration instance.
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| url | Server URL \(e.g., https://example.net\) | True |
+| credentials | Username | True |
+| isFetch | Fetch incidents | False |
+| max_fetch | Max fetch | False |
+| fetch_time | First fetch timestamp \(&lt;number&gt; &lt;time unit&gt;, e.g., 12 hours, 7 days, 3 months, 1 year\) | False |
+| score | CyberArk PAS score \(0.0\-100.0\) | False |
+| incidentType | Incident type | False |
+| insecure | Trust any certificate \(not secure\) | False |
+| proxy | Use system proxy settings | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+## Commands
```

Comment:
Before the commands, is there anything a customer should know regarding the setup/permissions needed?

Reply:
not that I know of, our instance is no the "traditional" one.